### ISSUE TITLE
Library tests failing

### ISSUE THREAD

**Issue by ivanbarsukov** [semiautomaticgit/remotior_sensus#3]
Hello,

I recently attempted to build remotior_sensus and run the tests and found that a majority of them failed. Specifically, only 7 out of 32 tests passed. This is concerning as it suggests potential issues with the library's functionality and reliability.

Could you please confirm if these tests are expected to be operational at this stage? If so, I would appreciate any guidance on possible causes for these failures.

Here's a brief overview of my environment:

- Operating System: Archlinux
- Python Version: 3.11.6
- Library Version:  0.2.01

Any insights or assistance you can provide in resolving these test failures would be greatly appreciated.

Thank you.

**Comment by ivanbarsukov** [comment]
Tests status
![image](https://github.com/semiautomaticgit/remotior_sensus/assets/14816075/6ec50b35-6836-4315-9169-a5c67c3dbf88)

**Comment by semiautomaticgit** [comment]
Hello @ivanbarsukov , all the tests are operational.
Probably there are installation issues with the dependencies. 
Please copy the log file.

**Comment by ivanbarsukov** [comment]
Well, I found the test failure problem. The test code uses relative paths for files in "data" and assumes that "unittest" is run from within the "tests" directory. I think it would be great if this were fixed as it would allow for example to run them from the IDE.
If it's possible to make path-related errors more obvious, rather than TypeError, KeyError, etc inside the code, it would be faster to identify the problem in the future. But I think this may require more refactoring.
